data/libs/prelude.icicle

```diff
@@ -37,7 +40,7 @@ snd x = case x | (a,b) -> b end.
 group_keys k = keys (group k ~> 0).
 
 # This only works if inlining is set to with-subst (:set inline with-subst in the REPL)
-group_vals k v = vals (group k ~> v).
+# group_vals k v = vals (group k ~> v).
```

Comment:
ah righto, perhaps we should remove it altogether then, I'm not sure what we gain by having it in there commented out?